<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reporter</class>
 <widget class="QMainWindow" name="Reporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reporter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:window {          /* Borders around the code editor and debug window */
  border: 0px solid white;
  background-color: white;
}

QToolTip {
  background-color: white;
  color: black;
    padding: 5px;
    border-radius: 0;
    opacity: 200;
}

/* ==================== Dialog ==================== */
QLabel {
  background: white;
  color: black; 
  font: 75 11pt &quot;DejaVu Sans Mono&quot;;     /* Not sure about this one */
}

QDialog, QListView {
  background-color: white;
  color: black;
  outline: 0;
  border: 2px solid transparent;
}

QListView::item:hover {
  color: #F5F5F5;
  background: #F5F5F5;
}


QListView::item:selected {
  color: #F5F5F5;
  background: #F5F5F5;
}

/* === QTabBar === */
QTabBar {
  background: white;
}
QPlainTextEdit{
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}
QTableView{
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}
QScrollArea{
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}
QScrollArea{
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}


QTabWidget::pane {
  background: transparent;  /* Only at the very bottom of the tabs */
}

QTabBar::tab {
  background: white;
  border: 0px solid white;
  border-bottom: 2px solid white;
  color: black;
  padding-left: 10px;
  padding-right: 10px;
  padding-top: 3px;
  padding-bottom: 3px;
}

QTabBar::tab:hover {
  background-color: #F5F5F5;
  border: 0px solid #F5F5F5;
  border-bottom: 2px solid #F5F5F5;
  color: black;
}

QTabBar::tab:selected {
  background-color: #F5F5F5;
  border: 0px solid #F5F5F5;
  border-top: none;
  border-bottom: 2px solid #F5F5F5;
  color: black;
}

QStackedWidget {
  background: white;  /* This covers a bunch of things, I was thinking about making it transparent, */
              /* but I would have to find all the other elements... but QTabWidget::pane may be it */
}


/* === QGroupBox === */
QGroupBox {
    border: 1px solid white;
    background-color: white;
    margin-top: 1em;
}

QGroupBox::title {
  color: black;
    subcontrol-origin: margin;
    left: 6px;
    padding: 0 3px 0 3px;
}

QComboBox {
  color: black;
  background-color: white;
  selection-background-color: #F5F5F5;
  outline: 0;
}

QComboBox QAbstractItemView
{    
    selection-background-color: #F5F5F5;
  outline: 0;
}

/* === QCheckBox === */
QCheckBox, QRadioButton {
  color: black;
}

QCheckBox::indicator::unchecked  {
  background-color: white;
  border: 1px solid white;
}

QRadioButton::indicator::unchecked {
  background-color: white;
  border: 1px solid white;
  border-radius: 4px;
}

QCheckBox::indicator::checked, QTreeView::indicator::checked {
  background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #80CBC4, stop:1 white);
  border: 1px solid white;
}

QRadioButton::indicator::checked {
  background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #80CBC4, stop:1 white);
  border: 1px solid white;
  border-radius: 4px;
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled, QTreeView::indicator:disabled {
  background-color: white;      /* Not sure what this looks like */
}

QCheckBox::indicator::checked:disabled, QRadioButton::indicator::checked:disabled, QTreeView::indicator::checked:disabled {  
  background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #BBBBBB, stop:1 white); /* Not sure what this looks like */
}

QTreeView {
  background-color: white;
  color: black;
  outline: 0;
  border: 0;
}

QTreeView::item:hover {
  background-color: white;
  color: black;
}

QTreeView::item:selected {
  background-color: #F5F5F5;
  color: black;
}

QTreeView QHeaderView:section {
  background-color: white;
  color: black;
  border: 0;
}

QTreeView::indicator:checked {
  background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #80CBC4, stop:1 white);
  border: 1px solid #F5F5F5 ;
  selection-background-color: #F5F5F5 ;
}

QTreeView::indicator:unchecked {      /* This and the one above style the checkbox in the Options -&gt; Keyboard */
  background-color: white;        /* This is still a hover over in blue I can't get rid of */
  border: 1px solid white;
  selection-background-color: white;
}
/*
QTreeView QScrollBar {
  background-color: #F5F5F5
}
*/
QTreeView::branch {
  /* Skip - applies to everything */
}

QTreeView::branch:has-siblings:adjoins-item {
  /* Skip - files */
}

QTreeView::branch:has-siblings:!adjoins-item {
  /* Skip - applies to almost all on the left side */
}

QTreeView::branch:closed:has-children:has-siblings {
  background: url('./images/rightarrowgray.png') center center no-repeat;
}

QTreeView::branch:has-children:!has-siblings:closed {
  background: url('./images/rightarrowgray.png') center center no-repeat;
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
  /* Skip - files */
}

QTreeView::branch:open:has-children:has-siblings {
  background: url('./images/downarrowgray.png') center center no-repeat;
}

QTreeView::branch:open:has-children:!has-siblings {
  background: url('./images/downarrowgray.png') center center no-repeat;
}

/* === QScrollBar:horizontal === */
QScrollBar:horizontal {
  background: #f44336;        /* Background where slider is not */
  height: 10px;
  margin: 0;
}

QScrollBar:vertical {
  background: #f44336;        /* Background where slider is not */
  width: 10px;
  margin: 0;
}

QScrollBar::handle:horizontal {
    background: #f44336;          /* Slider color */
    min-width: 16px;
  border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: #f44336;          /* Slider color */
    min-height: 16px;
  border-radius: 5px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal,
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
  background: #f44336;                        /* Removes the dotted background */
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal,
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {  /* Hides the slider arrows */
      border: #f44336;
      background: #f44336;
}

QPushButton {
  background-color: #f44336;
  color: black;
  border: 1px solid #f44336;
  padding: 4px 22px;
}

QPushButton:hover {
  color: #F5F5F5;
}

QPushButton:pressed {
  color: #FFFFFF;
}

QLineEdit {
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}

QSpinBox {
  background: #F5F5F5;
  border: 1px solid #F5F5F5;
  color: black;
}

/*****************************************************************************
Main Screen
*****************************************************************************/
QTreeView {
  background-color: white;
}

QMenu {
  background-color: #f44336;    /* File Menu Background color */
  color: black;
}

QMenu::item:selected {
  color: #F5F5F5;
}

QMenu::item:pressed {
  color: #FFFFFF;
}

QMenu::separator {
  height: 1px;
  background: transparent;      /* Could change this to #546E7A and reduce the margin top and bottom to 1px */
  margin-left: 10px;
  margin-right: 10px;
  margin-top: 5px;
  margin-bottom: 5px;
}

/* === QMenuBar === */
QMenuBar {
  background-color: white;
  color: black;
}

QMenuBar::item {
  background: transparent;
}

QMenuBar::item:disabled {
  color: gray;
}

QMenuBar::item:selected {
  color: #F5F5F5;
}

QMenuBar::item:pressed {
  color: #FFFFFF;
}

QToolBar {
  background: #f44336;
  border: 1px solid transparent;
}

QToolBar:handle {
  background: transparent;
  border-left: 2px dotted #80CBC4;  /* Fix the 4 handle dots so it doesn't look crappy */
  color: transparent;
}

QToolBar::separator {
  border: 0;
}

/* === QToolButton === */
QToolButton:hover, QToolButton:pressed {
  background-color: #F5F5F5;
}

QToolButton::menu-button {
  background: url('./images/downarrowgray.png') center center no-repeat;
  background-color: #f44336;                        
}

QToolButton::menu-button:hover, QToolButton::menu-button:pressed {
  background-color: #f44336;
}

QStatusBar {
  background-color: #f44336;
}

QLabel {
  color: black;
}

QToolButton {
  color: white;
  background-color: #f44336;
}

QToolButton:hover, QToolButton:pressed, QToolButton:checked {
  background-color: #F5F5F5;
}

QToolButton:hover {
  color: black;

}

QToolButton:checked, QToolButton:pressed {
  color: #FFFFFF;
}


QToolButton {
  border: 1px solid #f44336;
  margin: 1px;
}

QToolButton:hover {
  background-color: #F5F5F5;
  border: 1px solid #F5F5F5;
}

QToolButton[popupMode=&quot;1&quot;] {
  padding-right: 20px;
}

QToolButton::menu-button {
  border-left: 1px solid #f44336;
  background: transparent;
    width: 16px;
}

QToolButton::menu-button:hover {
  border-left: 1px solid #f44336;
  background: transparent;
    width: 16px;
}

QStatusBar::item {
  color: white;
  background-color: #f44336;
}

QAbstractScrollArea  {  /* Borders around the code editor and debug window */
  border: 0;
}

/*****************************************************************************
Play around with these settings
*****************************************************************************/

/* Force the dialog's buttons to follow the Windows guidelines. */
QDialogButtonBox {
    button-layout: 0;
}

QTabWidget::tab-bar {
  left: 0px; /* Test this out on OS X, it will affect the tabs in the Options Dialog, on OS X they are centered */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1090</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="basic">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="dbWidget" native="true">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>321</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="dbName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TestDB</string>
       </property>
      </widget>
      <widget class="QToolButton" name="dbConnect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dbPassword">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="paramWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>621</width>
        <height>291</height>
       </rect>
      </property>
      <widget class="QToolButton" name="paramNew">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
      <widget class="QToolButton" name="paramEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="param1">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="param2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="param3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="param4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>210</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="param5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>251</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">	background:#F5F5F5;
	border: 1px solid #F5F5F5;
	color: black;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="scrollLayout_2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QToolButton" name="paramDelete">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SQL">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableView" name="queryTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>1061</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>410</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>generate</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="queryEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QToolButton" name="newQuery">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>new query</string>
      </property>
     </widget>
     <widget class="QToolButton" name="saveQuery">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>410</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>edit selected</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="queryNameEdit">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>60</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="queryParamEdit">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>100</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QToolButton" name="clearQuery">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>410</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>140</y>
        <width>241</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">	background:#F5F5F5;
	border: 1px solid #F5F5F5;
	color: black;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="scrollLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QToolButton" name="queryDelete">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>410</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
